### PR TITLE
Readme updated.dev note section added

### PR DESCRIPTION
**Pull request by mudaligek**
Hi, Please could you review my changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
+## Development Notes
+
 This is Readme for cc_mobile
```
Comment (mudaligekosala) [inline README.md:3]: Add More details to this section

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
+## Development Notes
```
Comment (mudaligekosala) [inline README.md:1]: Chane header to Dev

**Review by mudaligekosala — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mudaligek) [reply README.md:1]: Done

File: README.md (hunk 1 above)
Comment (mudaligek) [reply README.md:3]: Done

**Review by mudaligekosala — APPROVED** [review]

**Pull request merged**